src/EventSource.js

```diff
@@ -58,53 +61,62 @@ define(function() {
         return match;
     }
 
-    /**
-     * Static functions that provide an easy way to tell what type of event you have.
-     *
-     * If you are wondering why this file looks like it has no coverage in Istanbul, it is because
-     * most of the unit tests do not run in phantom.
-     */
-    var EventSource = {};
-
-    EventSource.isPointer = function (event) {
-        // PointerEvent is IE11, MSPointerEvent is IE10.
-        return checkEventType(event, window.PointerEvent) || checkEventType(event, window.MSPointerEvent);
-    };
-
-    EventSource.isMouse = function (event) {
+    function isPointerMouseOrPen(event) {
         var result = false;
-        var pType = event.pointerType;
 
         if (EventSource.isPointer(event)) {
-            // Treat MS pointer events with types of mouse and pen as mouse events.
+            var pointerEvent = getBrowserEvent(event);
+            var pType = pointerEvent.pointerType;
+
             if (pType === 'mouse' || pType === MSPOINTER_TYPE_MOUSE ||
                 pType === 'pen' || pType === MSPOINTER_TYPE_PEN) {
                 result = true;
             }
-        } else {
-            result = checkEventType(event, window.MouseEvent);
         }
 
         return result;
-    };
+    }
 
-    EventSource.isTouch = function (event) {
+    function isPointerTouch(event) {
         var result = false;
 
         if (EventSource.isPointer(event)) {
-            // Treat MS pointer events with types of touch as touch events.
-            if (event.pointerType === 'touch' || event.pointerType === MSPOINTER_TYPE_TOUCH) {
+            var pointerEvent = getBrowserEvent(event);
+            var pType = pointerEvent.pointerType;
+
+            if (pType === 'touch' || pType === MSPOINTER_TYPE_TOUCH) {
                 result = true;
             }
-        } else {
-            result = checkEventType(event, window.TouchEvent);
         }
 
         return result;
+    }
+
+    /**
+     * Static functions that provide an easy way to tell what type of event you have.
+     *
+     * If you are wondering why this file looks like it has no coverage in Istanbul, it is because
+     * most of the unit tests do not run in phantom.
+     */
+    var EventSource = {};
+
+    EventSource.isPointer = function(event) {
+        // PointerEvent is IE11, MSPointerEvent is IE10.
+        return isEventOfType(event, window.PointerEvent) || isEventOfType(event, window.MSPointerEvent);
```

Comment:
A PointerEvent can have a pointer type of mouse, pen or touch. isPointer is a more general classification, whereas isPointerMouseOrPen and isPointerTouch is a subset of those things that would match isPointer.